# ProcessState.srv
Array desired_state
Array weighting
---
Array processed_state